### PR TITLE
docs: ✏️ mention the Training metrics tab

### PR DESCRIPTION
**Pull request by severo**
✅ Closes: #190

### CODE REVIEW

**Comment by severo** [comment]
BTW I have no idea about how to solve the CI failure, but I don't think it's linked to this PR

File: docs/hub/adding-a-model.md [hunk 1]
```diff
@@ -104,3 +104,11 @@ You can add metadata to your model card. You can specify:
 * a lot more!
 
 Read more about model tags [here](https://huggingface.co/docs/hub/model-repos#how-are-model-tags-determined).
+
+8. Add TensorBoard traces
+
+Models trained with 🤗 Transformers will generate [TensorBoard traces](https://huggingface.co/transformers/main_classes/callback.html?highlight=tensorboard#transformers.integrations.TensorBoardCallback) by default if [`tensorboard`](https://pypi.org/project/tensorboard/) is installed.
+
+Beyond 🤗 Transformers, any repository that versions TensorBoard traces (filenames that contain `tfevents`) is categorized with the [`TensorBoard` tag](https://huggingface.co/models?filter=tensorboard). The "Training metrics" tab then makes it easy to review charts of the logged variables, like the loss or the accuracy.
```
Comment (julien-c) [inline docs/hub/adding-a-model.md:112]:
```suggestion
Beyond 🤗 Transformers, any repository that contains TensorBoard traces (filenames that contain `tfevents`) is categorized with the [`TensorBoard` tag](https://huggingface.co/models?filter=tensorboard). As a convention we suggest that you save traces under the `runs/` subfolder. The "Training metrics" tab then makes it easy to review charts of the logged variables, like the loss or the accuracy.
```

**Review by julien-c — APPROVED** [review]
small suggestion and Github won't let me see the image for some reason, but otherwise 🔥

**Comment by severo** [comment]
OK, thanks

**Pull request merged**